docs/discovery/bos/components.md

```diff
@@ -0,0 +1,132 @@
+---
+sidebar_position: 3
+---
+
+# Components
+
+## Overview
+
+Components are a fundamental aspect of the Blockchain Operating System, providing a standard unit of software that packages up code and dependencies for rapid and reliable execution. These standalone, lightweight, and reusable components store HTML/CSS/JavaScript code on-chain and can be loaded into gateways for users.
+
+You can view and create components at [bos.gg](https://bos.gg).
+
+#### Programming language
+Components are coded in a limited version of JSX (JavaScript with ReactJS) and are stored on-chain, where they are loaded and executed in a custom Virtual Machine. If you are unfamiliar with ReactJS, this [React tutorial](https://reactjs.org/tutorial/tutorial.html) should be helpful.
+
+#### Open source with version control
+Each component’s source code is stored on a public blockchain, making them open source by default. Component owners can upgrade components by updating the source code. Since the code is stored on-chain, all previous versions of a component are also available. This enables automatic version control for every component. 
+
+#### Forking
+If you are familiar with Git, you are probably familiar with the concept of forking, which enables users to fork any public repository to create their own versions. Similarly, you can fork any BOS component to get started quickly. For example, the Lido[URL] component is a great starting point.
+
+#### Deploying
+The source code of each component is stored on-chain. BOS uses the NEAR blockchain underneath. Deployment is easy, with just a few clicks. To deploy a component, you will need a NEAR account and NEAR tokens for the storage deposit. 
+
+## Getting started
+
+To begin building a component, we recommend following our tutorial as the best way to get started. The first two tutorials listed below are quick and easy ways to get familiar with building with BOS, taking less than 5 minutes each. In addition, the Lido example provides a fully-fledged demonstration of how to interact with a smart contract deployed on the Ethereum mainnet.
+
+* [Hello World](Tutorials/hello-world.md)
+* [Hello EthersJS](Tutorials/hello-ethersJS.md)
+* [Hello Lido](Tutorials/hello-lido.md)
```

Comment:
The last two (EtherJS and Lido) I don't know if we have them. We should bring them into our bos tutorial section

Reply:
perfect